exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/SystemTable.java

```diff
@@ -31,6 +31,7 @@
  * </p>
  */
 public enum SystemTable {
+  @Deprecated
```

Comment:
Why we need to deprecate?

Reply:
I want to actually replace the sys.options completely with sys.options_val , but I broke tests. Hoping to discourage the usage this way.

Reply:
It should really be the other way around; i.e deprecate sys.options_val and retain sys.options. Options_val coalesces the xxx_value columns to show only one value column which is desirable. But it also shows only the value in effect for the user which IMO is less useful. 
@amansinha100 had a good suggestion which was to have one large sys.options table but provide appropriate views for different use cases.

Reply:
I don't think the challenge is so much in deprecating either. I actually have a commit that renames the two tables as `sys.options_old` and s`ys.options`. The concern seems to be around breaking backward compatibility tests.
Are you suggesting we show default values in addition to changed value??

Reply:
I see inconsistent behaviour for sys.options. If I override an option at the session level, two rows show up in the table, one for the default and one for the overridden (sys.options_val always shows only one). If, however, I override an option at the system level only one row shows up in the table, the overridden one. 
Ideal would be to show both, but also coalesce the columns for the value (as in options_val).
I think we should log a separate Jira for that.

Reply:
Interesting. I've never seen that whenever I've modified at a session level. Could you file a JIRA for that?

Reply:
The bug is not from this commit, so will ignore this for now. 
Should I leave this deprecated or remove it?

Reply:
What is the decision on this part?

Reply:
I've removed the deprecated flag for now. I logged [DRILL-6684](https://issues.apache.org/jira/browse/DRILL-6684)  for this purpose.